packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -14,6 +14,9 @@ import 'package:meta/meta.dart';
 import 'assertions.dart';
 import 'basic_types.dart';
 import 'constants.dart';
+import 'core_stub.dart'
```

Comment:
Per my other comment, this would also change to just `import 'core.dart';`

Reply:
+1 on @jakemac53's suggestions on this naming convention for conditional imports.